dev/com.ibm.ws.jaxrs.2.0_fat/publish/servers/com.ibm.ws.jaxrs.fat.beanparam/server.xml

```diff
@@ -8,4 +8,5 @@
     <!-- <webContainer enablemultireadofpostdata="true" /> -->
   	<include location="../fatTestPorts.xml"/>
   	<javaPermission className="java.util.PropertyPermission" name="bvt.prop.HTTP_default" actions="read"/>
+  	<javaPermission className="java.net.SocketPermission" name="*" actions="resolve"/>
```

Comment:
Why do we need these new permissions?  Is RESTEasy making a call that should be in a doPriv?  If so, we shouldn't add these permissions; we should add a doPriv where needed.

Reply:
I added a doPriv to address the problem and created a RestEasy jira:  https://issues.redhat.com/browse/RESTEASY-2707